storybook/stories/API/cartesian/Line.stories.tsx

```diff
@@ -5,22 +5,358 @@ import { expect } from '@storybook/jest';
+            '<Line dataKey="value" activeDot={<CustomizedDot />} />\n' +
+            '<Line dataKey="value" activeDot={renderDot} />',
+        },
+        defaultValue: true,
+        category: 'Responsive',
+      },
+    },
+    animateNewValues: { table: { category: 'Animation' } },
+    animationBegin: {
+      description: 'Specifies when the animation should begin, the unit of this option is ms.',
+      type: { name: 'number' },
+      defaultValue: 0,
+      table: { category: 'Animation' },
+    },
+    animationDuration: {
+      description: 'Specifies the duration of animation, the unit of this option is ms.',
+      type: { name: 'number' },
+      defaultValue: 1500,
+      table: { category: 'Animation' },
+    },
+    animationEasing: {
+      description: 'The type of easing function.',
+      type: { name: 'ease | ease-in | ease-out | ease-in-out | linear' },
+      defaultValue: 'ease',
+      table: { category: 'Animation' },
+    },
+    animationId: { table: { category: 'Animation' } },
+    baseLine: { table: { category: 'Other' } },
     connectNulls: {
-      control: {
-        type: 'boolean',
+      description: 'Whether to connect a graph line across null points.',
+      table: {
+        type: {
+          summary: 'boolean',
+        },
+        category: 'Data',
       },
+      defaultValue: false,
     },
-    stroke: {
-      control: { type: 'color' },
+    dangerouslySetInnerHTML: { table: { category: 'Other' } },
+    data: { table: { category: 'Data' } },
+    dataKey: {
+      description:
+        'The key or getter of a group of data which should be unique in a LineChart. It could be an accessor function such as (row)=>value',
+      table: {
+        type: { summary: 'string | number | function' },
+        category: 'Data',
+      },
+    },
+    dot: {
+      description:
+        'If false set, dots will not be drawn. If true set, dots will be drawn which have the props calculated internally. If object set, dots will be drawn which have the props mergered by the internal calculated props and the option. If ReactElement set, the option can be the custom dot element.If set a function, the function will be called to render customized dot.',
+      table: {
+        type: {
+          summary: 'Boolean | Object | ReactElement | Function',
+          detail:
+            '<Line dataKey="value" dot={false} />\n' +
+            '<Line dataKey="value" dot={{ stroke: \'red\', strokeWidth: 2 }} />\n' +
+            '<Line dataKey="value" dot={<CustomizedDot />} />\n' +
+            '<Line dataKey="value" dot={renderDot} />',
+        },
+        category: 'Style',
+        defaultValue: true,
+      },
     },
     fill: {
       control: { type: 'color' },
+      table: { category: 'Style' },
+    },
+    hide: {
+      description: 'Hides the line when true, useful when toggling visibility state via legend',
+      type: { name: 'boolean' },
+      defaultValue: false,
+      table: { category: 'Style' },
```

Comment:
nit: general comment - should we put all properties that are in the same category adjacent to eachother in this JSON? Might help just for code organization

Reply:
Good point.

Reply:
I improved the organisation. For some props I am not certain whether my suggested category makes sense. We will most likely iterate here as we add more ArgsTables.